### PR TITLE
44 jh meme dumpster fire

### LINKED ISSUE #44: Add dumpster fire
its fine

### PR DESCRIPTION
**Pull request by jacobhoffmann257**
closes #44

### CODE REVIEW

**Review by armstrca — APPROVED** [review]

**Review by thatcantbeso — DISMISSED** [review]
Close these extra lines pls

File: README.md [hunk 1]
```diff
@@ -10,10 +10,14 @@ Clone this repository (or open in codespace). Create an issue with a meme descri
 | -------- | ------- | ------- |
 | ![](./assets/wheres-the-mouse.webp) | [Ian Heraty](https://github.com/heratyian) | [source](https://wallpapers.com/images/featured-full/cat-meme-pictures-9anza1hu2l2k0wyp.jpg) |
 | ![](./assets/css_pro.webp) | [Mena Castillo](https://github.com/Meenoow) | [source](https://medium.com/agileactors/program-memes-45f69695457e) |
+|![](./assets/dumpster.webp)| [Jacob Hoffmann](https://github.com/jacobhoffmann257) | [source](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExa25zcTlocTdpdTNvc3hubHk3enJuOG51Zmh1dHoweGpuYXZpZ3psbSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9cw/giLKqcYUzgoUe9tioR/giphy.gif)
+| ![](./assets/pickle-meme-aldo.jpg) | [Aldo Portillo](https://github.com/aldoportillo) | [source](https://pbs.twimg.com/media/FsXbMgoWcAAut83.jpg) |
+
```
Comment (thatcantbeso) [inline README.md]: delete

File: README.md [hunk 2]
```diff
@@ -10,10 +10,14 @@ Clone this repository (or open in codespace). Create an issue with a meme descri
...
 | ![](./assets/one-does-not-zjxkyt.jpg)| [Andrew Cook](https://github.com/andrewhcook) | [source](https://media.makeameme.org/created/one-does-not-zjxkyt.jpg) |
 | ![](./assets/Oprah-meetings-meme-1.png) | [Alex Mompremier](https://github.com/ajmompr) | [source](https://www.ntaskmanager.com/wp-content/uploads/2018/03/Oprah-meetings-meme-1.png) |
 | ![](./assets/coffee-meme-hello-darkness-my-old-friend.jpg) | [Fidel Leal](https://github.com/fidel-techprep) | [source](https://i0.wp.com/roomescapeartist.com/wp-content/uploads/2016/07/coffee-meme-hello-darkness-my-old-friend.jpg?ssl%253D1) |
 | ![](./assets/pickle-meme-aldo.jpg) | [Aldo Portillo](https://github.com/aldoportillo) | [source](https://pbs.twimg.com/media/FsXbMgoWcAAut83.jpg) |
+
```
Comment (thatcantbeso) [inline README.md]: delete

**Review by melo616 — APPROVED** [review]

**Review by thatcantbeso — APPROVED** [review]
beautiful

**Pull request merged**